other/maximum_subarray.py

```diff
@@ -21,6 +21,6 @@ def max_subarray(nums: list[int]) -> int:
 
 
 if __name__ == "__main__":
-    n = int(input("Enter number of elements : ").strip())
-    array = list(map(int, input("\nEnter the numbers : ").strip().split()))[:n]
-    print(max_subarray(array))
+    array = list(map(int, input("\nEnter the numbers : ").strip().split()))
+    n = len(array)
+    print(max_subarray(array[:n]))
```

Comment:
Whats the point of doing `array[:n]` or even creating variable `n`? All you're doing is saying "index this list up to all its elements"

Reply:
yea, You are right, n is not need at all